### ISSUE TITLE
Project not synchronize with NPM

### ISSUE THREAD

**Issue by AlexandreCantin** [chainlist/svelte-forms#17]
Hi,

I've downloaded svelte-forms for a personal project directly from NPM and got the version : `1.2.0`.

Having a bug in this version with async validations, I have found the problem and fixed it in`index.js` :
```
if (!valid) {
    n.fields[fieldName].errors.push(rule); // <== should be name 
}
```

Then, I have planned to fix directly in this repo but I found two problems :
 - my bug is already fixed (not a problem by the way ^^)
 - but the `package.json` refers to the version 1.0.1 and there is no reference to the version 1.2.0 from NPM 🤔 

In conclusion, I have no idea how to deal with this situation... => The NPM package seems late compared to the repo but if I look the package version, it's the contrary :/ 

=> Time to reboot to an identical version 1.3.0 on NPM and on Github...? 😉

Thanks in advance for your answer,
Alexandre

**Comment by dzaghal** [comment]
Hello,

I am in the same situation. 

I wanted to propose a little update (adding 'dirty' class for every field in form), but again my fork from npm is different.

It's a pity because the change is quite trivial.

Can this be solved?

Thank you.
Ferran

**Comment by chainlist** [comment]
Hi @AlexandreCantin,

Sorry for the late answer, life gone crazy lately!
Indeed last version hasn't been published on NPM.
Going to do this very soon.

**Comment by chainlist** [comment]
PR has been merged and NPM package reflects now the github states
